api/authorization.rst

```diff
@@ -43,25 +45,47 @@ __ https://github.com/commerceguys/guzzle-oauth2-plugin
 Obtain access token
 ~~~~~~~~~~~~~~~~~~~
 
-Send the request with the following parameters
+Send the request with the following parameters:
+
+Definition
+..........
 
-client_id
-    Public client id
-client_secret
-    Client secret
-grant_type
-    We will use 'password' to authorize as user
-username
-    User name
-password
-    User password
+.. code-block:: text
+
+    GET /oauth/v2/token
```

Comment:
We can also you `POST` here. Not sure was is a better practice, but maybe we should at least mention about the `POST`?